in_toto/model.go

```diff
@@ -898,14 +900,26 @@ func (mb *Metablock) VerifySignature(key Key) error {
 		return err
 	}
 
-	dataCanonical, err := mb.GetSignableRepresentation()
+	verifier, err := getSignerVerifierFromKey(key)
 	if err != nil {
 		return err
 	}
 
-	if err := VerifySignature(key, sig, dataCanonical); err != nil {
+	payload, err := mb.GetSignableRepresentation()
+	if err != nil {
+		return err
+	}
+
+	sig_bytes, err := hex.DecodeString(sig.Sig)
```

Comment:
```suggestion
	sigBytes, err := hex.DecodeString(sig.Sig)
```

nit: consistency in variable naming

Reply:
Updated as requested.